package.json

```diff
@@ -364,7 +364,7 @@
     "@opentelemetry/resources": "^1.4.0",
     "@opentelemetry/sdk-metrics-base": "^0.30.0",
     "@opentelemetry/semantic-conventions": "^1.4.0",
-    "@reduxjs/toolkit": "^1.6.1",
+    "@reduxjs/toolkit": "1.7.2",
```

Comment:
can we keep `^`?

Reply:
Done

Reply:
Doesn't look like you pushed the commit when the fix

Reply:
I just removed it again to fix typing issues elsewhere. We cannot upgrade to 1.8.5 right now, because the ReduxEmbeddable code in the Embeddable plugin breaks on the 1.8 line. So need to keep this at 1.7.2, at least for now.